tests/lib/prepare-restore.sh

```diff
@@ -740,6 +740,12 @@ restore_suite() {
 }
 
 restore_project_each() {
+    $STRAY_TASKS=$(cat /var/lib/snapd/state.json | jq '.tasks[] | select (.change == "") | {"id":.id,"summary":.summary,"kind":.kind}')
+    if [ $(echo "$STRAY_TASKS" | wc -l) -gt 0 ]; then
+        echo "$STRAY_TASKS"
```

Comment:
could we make the message slightly more clear as in

```suggestion
        echo "there are stray tasks without an associated change leftover in the state.json:"
        echo "$STRAY_TASKS"
```